Requests to localhost stop working in 2.2.0

# Bug Report

## Description
After updating to 2.2.0 requests to localhost are not working as before.

## To Reproduce
Steps to reproduce the behavior:

1. Start a server on localhost that responds with status 200 on port 8420 (or any other)
2. Run this JUnit test case with 2.2.0:

```Kotlin
  @Test
    fun testServerWorks() {
        val (_, response, _) = "http://localhost:8420".httpGet().responseString()
        Assertions.assertThat(response.statusCode).isEqualTo(200)
    }
```

## Expected behavior
The test passes just like it does when I change the dependency back to Fuel 2.1.0.

## Actual behavior
The test fails with status code being -1: `org.junit.ComparisonFailure: Expected :200 Actual   :-1`.
If I poke around a bit, I see a FuelError thrown in RequestTask.kt:24 and the result is this:
```
[Failure: connection.responseMessage must not be null
	com.github.kittinunf.fuel.core.FuelError$Companion.wrap(FuelError.kt:85)
	com.github.kittinunf.fuel.core.DeserializableKt.response(Deserializable.kt:168)
	com.github.kittinunf.fuel.core.requests.DefaultRequest.responseString(DefaultRequest.kt:475)
	com.example.demo.DemoApplicationTests2.testServerWorks(Test2.kt:11)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.base/java.lang.reflect.Method.invoke(Method.java:566)
	org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: connection.responseMessage must not be null
	com.github.kittinunf.fuel.core.FuelError$Companion.wrap(FuelError.kt:86)
	com.github.kittinunf.fuel.core.requests.RequestTask.executeRequest(RequestTask.kt:24)
Caused by: java.lang.IllegalStateException: connection.responseMessage must not be null
	com.github.kittinunf.fuel.toolbox.HttpClient.retrieveResponse(HttpClient.kt:205)
	com.github.kittinunf.fuel.toolbox.HttpClient.doRequest(HttpClient.kt:79)
	com.github.kittinunf.fuel.toolbox.HttpClient.executeRequest(HttpClient.kt:36)
	com.github.kittinunf.fuel.core.requests.RequestTask.executeRequest(RequestTask.kt:23)
	com.github.kittinunf.fuel.core.requests.RequestTask.call(RequestTask.kt:44)
	com.github.kittinunf.fuel.core.requests.RequestTask.call(RequestTask.kt:14)
	com.github.kittinunf.fuel.core.DeserializableKt.response(Deserializable.kt:166)
	com.github.kittinunf.fuel.core.requests.DefaultRequest.responseString(DefaultRequest.kt:475)
	com.example.demo.DemoApplicationTests2.testServerWorks(Test2.kt:11)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.base/java.lang.reflect.Method.invoke(Method.java:566)
	org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)

]
```
## Environment

### Development Machine

 - OS: Ubuntu 18.04
 - IDE: IntelliJ 2019.2
 - Fuel version: 2.2.0
 - Kotlin version: 1.3.40

--- a/fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt
+++ b/fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt
@@ -202,7 +202,7 @@ class HttpClient(
             headers = headers,
             contentLength = contentLength ?: -1,
             statusCode = connection.responseCode,
-            responseMessage = connection.responseMessage,
+            responseMessage = connection.responseMessage.orEmpty(),
             body = DefaultBody.from(
                 { progressStream.buffered(FuelManager.progressBufferSize) },
                 { contentLength ?: -1 }

